src/test/java/com/digitalasset/testing/junit4/SandboxTest.java

```diff
@@ -0,0 +1,30 @@
+package com.digitalasset.testing.junit4;
+
+import org.junit.Assert;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExternalResource;
+
+import java.nio.file.Paths;
+
+public class SandboxTest {
+
+  private static Sandbox sandbox =
+      Sandbox.builder().dar(Paths.get("./src/test/resources/ping-pong.dar")).build();
+
+  @ClassRule public static ExternalResource classRule = sandbox.getClassRule();
+
+  @Rule public ExternalResource rule = sandbox.getRule();
+
+  @Test
+  public void getSandboxPort() {
+    int sandboxPort = sandbox.getSandboxPort();
+    assertAreBetween(sandboxPort, 6860, 6890);
```

Comment:
why not exactly 6865? maybe explain in the test name or a comment?

Reply:
You kinda have a valid point. I did not want to make a big redesign here, just expose the port and test it. Therefore I decided to do it as simply as possible, which was to hardcode the values of `com.digitalasset.testing.utils.SandboxUtils#SANDBOX_PORT_RANGE`.

Again, I do agree it is not the nicest option, however it is easy and works for now.

Reply:
Then maybe a comment or a more descriptive name will guide the reader not to ask silly questions ;)